### ISSUE TITLE
No way to restore backups easily. 

### ISSUE THREAD

**Issue by TylerCode** [lionelpere/azure-devops-repository-backup#12]
Just wanted to see if there is any interest in a restore script? I haven't integrated these scripts yet into the backup process, but I figure a restore script would probably be a good idea before I did. 

I can build the first iteration of it, but figured raising the issue would be a good idea first.

**Comment by lionelpere** [comment]
Yes but if you do it, you'll need to document the Archive structure.

**Comment by TylerCode** [comment]
You mean document the current Archive structure yeah? Not a problem.

**Comment by lionelpere** [comment]
Current or the required to restore data. 

I assumed you'd chose the same.

Thanks a lot for your ideas and contribution !

**Comment by lionelpere** [comment]
@claude Make some suggestion. Generate a plan to fix the issue.
